nfl_stats.py

```diff
@@ -9,8 +9,32 @@
     'DEF'
 )
 
-def get_player_points(player_name, week, position=None):
-
+class Player(object):
+
+    def __init__(self, info):
+        self.name = info['name']
+        self.pos = info['position']
+        self.points = self.calc_points(info)
+
+    def __repr__(self):
+        return "{0} {1}".format(self.name, self.points)
+
+    def calc_points(self, stats):
```

Comment:
was testing this and ran into an error, for stats i inputted the return value from gather_json and I used the GOAT QB but the points it returned was 0.0, please check your work